lib/logatron/logatron.rb

```diff
@@ -72,5 +74,14 @@ def msg_id
     def msg_id=(id)
       Logatron::Contexts.msg_id = id
     end
+
+    private
+
+    def exception_message(e, additional)
+      info = additional.is_a?(Hash) ? additional.map { |(k, v)| "#{k}=>#{v}"}.join(', ') : additional
+      info_msg = info.nil? || info.empty? ? '' : "; MORE_INFO( #{info} )"
```

Comment:
The reason for the interpolation is to log nothing extra if there is nothing to log. I do not think adding the text you suggested above is helpful in a log entry. It just makes more noise / space required.